handbook/product-design/README.md

```diff
@@ -56,7 +56,12 @@ At Fleet, like [GitLab](https://about.gitlab.com/handbook/product-development-fl
 
 3. Add page names (e.g. "Host details" page) to the user story's title and/or description to help contributors find Figma wireframes for the area of the UI you're making changes to.
 
-4. If the story requires API or YAML file changes, open a pull request to the reference docs release branch (e.g. `docs-v4.58.0`) with the proposed design. Mark the PR "Ready for review" as soon as it's ready for feedback from the [API design DRI](https://fleetdm.com/handbook/company/communications#directly-responsible-individuals-dris).
+4. If the story requires API or YAML file changes, open a pull request to the reference docs with the proposed design. Pay attention to existing conventions (URL structure, parameter names, response format) and aim to be consistent. Once your changes are ready, make a PR following these guidelines:
+  - Make a PR against the docs release branch for the version you expect this feature to be in. Docs release branches are named using the format `docs-vX.X.X`, so if you're designing for Fleet 4.61, you would make a PR to `docs-v4.61.0`.
+  - Add a link to the issue in the PR description.
+  - Attach the `~api-design` label.
+  - Mark the PR ready for review. (Draft PRs do not auto-request reviews.)
+  - After your changes are approved by the API design DRI and the design is settled, merge your changes into the docs release branch.
```

Comment:
```suggestion
  - After your changes are approved by the API design DRI, they will merge your changes into the docs release branch.
```

Instead of leaving it up to the Product Designer to merge, maybe the API design DRI merge in the changes when approved? 

- I think this would mean PRs wouldn't be open as long. API design DRI doesn't have to revisit an open PR just to see that it's already been approved.
- Also, I think it's up to the Product Designer to mark a PR as ready for review when it's ready for review i.e. "settled"